test/prompConfig.js

```diff
@@ -1,11 +1,32 @@
+import fs from 'fs-extra';
 import test from 'ava';
 import path from 'path';
 import chalk from 'chalk';
 import json from 'json-extra';
+import os from 'os';
 import getConfig from '../lib/getConfig';
 import { choices, questions } from '../lib/promptConfig';
 
 const cwd = process.cwd();
+const homedir = os.homedir();
+const randomString = Math.random().toString(36).replace(/[^a-z]+/g, '').substr(0, 10);
```

Comment:
`.back` is ~always a backup file, if something goes wrong we know that it is a backup